xml/art-obs-beginners-guide.xml

```diff
@@ -930,68 +965,83 @@ Fri Aug 23 12:31:41 UTC 2017 - &exampleuser_mail;</screen>
 Patch0:         &gitproject;_main.diff</screen>
    </step>
    <step>
-    <para>In the <literal>%prep</literal> section add the <literal>%patch</literal>
-     macro like this:</para>
+    <para>In the <literal>%prep</literal> section, add the <literal>%patch</literal>
+     macro:</para>
     <screen>%prep
 %setup -q -n %{name}-%{version}
 %patch0</screen>
    </step>
    <step>
     <para>
-     Add you patch file to the local repository and rebuild your package
-     with:
+     Add your patch file to the local repository and rebuild your package:
     </para>
     <screen>&prompt.user;<command>osc</command> add &gitproject;_main.diff
 &prompt.user;<command>osc</command> build openSUSE_Tumbleweed x86_64 *.spec</screen>
    </step>
    <step>
-    <para>If everything was successfull, commit your changes by adding
-     your commit message:
+    <para>
+     If everything was successful, commit your changes:
     </para>
     <screen>&prompt.user;<command>osc</command> commit</screen>
+    <para>
+     An editor will open, so you can specify and save a commit message.
+    </para>
    </step>
   </procedure>
   <para>
    If you have to deal with many patches, the tool
-   <systemitem>quilt</systemitem> could be useful.
+   <command>quilt</command> can be useful. For more information about
+   <command>quilt</command>, see
+   <link xlink:href="https://savannah.nongnu.org/projects/quilt"/>.
   </para>
+  <remark>
+   Homepage is admittedly not great, but did not want to link to Wikipedia. -
+   sknorr, 2017-08-16
+  </remark>
  </sect1>
 
  <sect1 xml:id="sec.obsbg.uc.branchprj">
-  <title>Branching a Project</title>
+  <title>Branching a Package</title>
+  <remark>
+   Is the OBS definition of branch really so narrow? (i.e. just for home
+   project copies?) - sknorr, 2017-08-16
+  </remark>
```

Comment:
@adrianschroeter question answered! my question referred to toms's definition of "branch" in the paragraph below.